superset/assets/src/SqlLab/components/QuerySearch.jsx

```diff
@@ -207,7 +207,7 @@ class QuerySearch extends React.PureComponent {
           <div className="col-sm-2">
             <AsyncSelect
               onChange={this.onChange}
-              dataEndpoint="/databaseasync/api/read?_flt_0_expose_in_sqllab=1"
+              dataEndpoint="/api/v1/databaseasync/?q=(filters:!((col:expose_in_sqllab,opr:eq,value:true))"
```

Comment:
@mistercrunch can you help, where is this used? I can make this call more efficient also, take a look at the other on. I mean we can only retrieve the keys and columns we want